roman-numerals.md

```diff
@@ -447,12 +447,8 @@ Here are some test cases, try and make them pass.
 {"50 gets converted to L", 50, "L"},
 ```
 
-If you're a cheater, all you needed to add to the `RomanNumerals` array is
+Need help? You can see what symbols to add in [this gist](https://gist.github.com/pamelafox/6c7b948213ba55332d86efd0f0b037de).
```

Comment:
When I was working through this, my window was scrolled such that the "cheater" section was already visible and it was hard not to look. Instead of accusing readers of being cheaters (which offended my fragile sensibilities), how about this approach of linking to a gist? I was going to link it to the code repository for this incremental version of the code, but it doesn't seem to exist.